### PR TITLE
Allow creating pebble format serving tables

### PR DESCRIPTION
**Pull request by tylerwilliams**
This change adds pebble support to the write_tables and http_server script.

### CODE REVIEW

File: kythe/go/serving/tools/write_tables/write_tables.go [hunk 1]
```diff
@@ -139,12 +140,23 @@ func main() {
 	}
 
 	if *compactTable {
```
Comment (jdhollen) [inline kythe/go/serving/tools/write_tables/write_tables.go:142]: Theres a call to compactleveldb at line 94, too. does that need to be updated?

File: kythe/go/storage/pebble/pebble.go [hunk 2]
```diff
@@ -122,6 +124,16 @@ func (p *pebbleDB) Get(ctx context.Context, key []byte, opts *keyvalue.Options)
 	return value, nil
 }
 
+func (p *pebbleDB) CompactRange(r *keyvalue.Range) error {
+	start := []byte{byte(0)}
+	end := []byte{byte(255)}
```
Comment (jdhollen) [inline kythe/go/storage/pebble/pebble.go:129]: what does this default value for start/end mean?? i'm struggling to parse how pebble uses these, but it seems like you should want this to mean "all keys"--i guess it's surprising to me that byte(255) would be lexicographically last?

**Review by jdhollen — APPROVED** [review]

File: kythe/go/serving/tools/write_tables/write_tables.go (hunk 1 above)
Comment (tylerwilliams) [reply kythe/go/serving/tools/write_tables/write_tables.go:142]: This write_tables script really has two codepaths: 1 that uses apache beam (flume basically) and one that does not. I've modifed the non-beam codepath so it supports pebble, but the beam one still requires more work due to how it writes the output files.

File: kythe/go/storage/pebble/pebble.go (hunk 2 above)
Comment (tylerwilliams) [reply kythe/go/storage/pebble/pebble.go]: Fixed.

**Review by tylerwilliams — COMMENTED** [review]
ptal

**Pull request merged**